lib/internal/navigator.js

```diff
@@ -126,7 +127,7 @@ class Navigator {
    * @return {string}
    */
   get platform() {
-    this.#platform ??= getNavigatorPlatform(process);
+    this.#platform ??= getNavigatorPlatform({ arch, platform });
```

Comment:
Can we make these parameters rather than an object? (For perf reasons)

Reply:
hmm - `getNavigatorPlatform` is an exposed function here, but it seems like it's only exposed for the purpose of unit testing. If that's the case, then absolutely, that's a great improvement and I'd be happy to make it.